src/awkward/_v2/contents/indexedarray.py

```diff
@@ -821,7 +821,7 @@ def _unique(self, negaxis, starts, parents, outlength):
             if isinstance(unique, ak._v2.contents.RegularArray):
                 unique = unique.toListOffsetArray64(True)
 
-            elif isinstance(unique, ak._v2.contents.ListOffsetArray):
+            if isinstance(unique, ak._v2.contents.ListOffsetArray):
```

Comment:
I had the same question about this as the next one, but I can see that they're symmetric bits of code.